internal/sink/sink.go

```diff
@@ -50,6 +60,27 @@ func NewSink(c Config, logger *zap.Logger, listener ancla.Register) Sink {
 			deliveryRetries:  c.DeliveryRetries,
 			logger:           logger,
 		}
+		return sink
+	case *ancla.RegistryV2:
+		var sinks Kafkas
+		for _, k := range l.Registration.Kafkas {
+			kafka := &Kafka{
+				id:         l.Registration.CanonicalName,
+				brokerAddr: k.BootstrapServers,
+				topic:      "test",
+			}
+
+			//TODO: this is basic set up for now - will need to add more options to config
+			//once we know what we are allowing users to send
+			config := sarama.NewConfig()
+			config.Producer.Return.Successes = true
+			config.Producer.RequiredAcks = sarama.WaitForAll
+			config.Producer.Retry.Max = c.DeliveryRetries //should we be using retryhint for this?
+
+			kafka.config = config
```

Comment:
we can add the producer here
`kafka.producer, err := sarama.NewSyncProducer(kafka.brokerAddr, kafka.config)`